xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -223,12 +274,22 @@
         <Parameter Name="byteOffset" Type="System.IntPtr" Index="1" FrameworkAlternate="dotnet-plat-ext-2.1;dotnet-plat-ext-2.2;net-5.0;net-6.0;netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">The type of reference.</typeparam>
-        <param name="source">The reference to add the offset to.</param>
+        <typeparam name="T">The elemental type of the managed pointer.</typeparam>
+        <param name="source">The managed pointer to add the offset to.</param>
         <param name="byteOffset">The offset to add.</param>
-        <summary>Adds a byte offset to the given reference.</summary>
-        <returns>A new reference that reflects the addition of byte offset to pointer.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Adds a byte offset to the given managed pointer.</summary>
+        <returns>A new managed pointer that reflects the addition of byte offset to the source pointer.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+The `byteOffset` parameter is the number of bytes to add to the `source` pointer. For example, given a source pointer _ptr_ of type `ref int`, the call `Unsafe.AddByteOffset<int>(ref ptr, 20)` will return a new pointer whose address points 20 bytes beyond _ptr_.
```

Comment:
```suggestion
The `byteOffset` parameter is the number of bytes to add to the managed `source` pointer. For example, given a managed source pointer _ptr_ of type `ref int`, the call `Unsafe.AddByteOffset<int>(ref ptr, 20)` will return a new managed pointer whose address points 20 bytes beyond _ptr_.
```
Is this overkill to add _managed_ to pointer? 
To distinguish between native- / raw-pointers (`void*`) and managed pointers (`ref T`) here?
Out of the context it's very clear what is meant, and the APIs are targeting advanced users, so this shouldn't be needed.
But as docs they should be very precise, even for people that don't know these differences and to avoid potential confusion.

This would apply to other places as well.